### PR TITLE
fix(ui): markdown

### PR DESCRIPTION
**Pull request by luzzif**

### CODE REVIEW

**Comment by Diogomartf** [comment]
Markdown not picking font family 

image

File: packages/ui/src/utils/components/index.tsx [hunk 1]
```diff
@@ -29,3 +29,10 @@ export function mergedCva<T extends object>(
         return twMergeWithConfig(classList(...classListParams));
     };
 }
+
+export function mergedCx(
```
Comment (Diogomartf) [inline packages/ui/src/utils/components/index.tsx:33]: not sure about the name of this function 🤔

Comment (luzzif) [reply packages/ui/src/utils/components/index.tsx:33]: It's the `cx` function with TW merge applied. It's following the same name conventions as `mergedCva`

**Comment by luzzif** [comment]
> Markdown not picking font family
> 
> image

I'm actually seeing it right now, can you retest?

**Review by Diogomartf — APPROVED** [review]

**Pull request merged**